locust/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+locust -f locustfile.py \
+  --host "${HOST}" \
+  --user "${NUM_USERS}" \
+  --spawn-rate "${SPAWN_RATE}" \
+  --headless \
+  --run-time "${RUN_TIME}" \
```

Comment:
Should any of these have defaults? I'm assuming Locust will exit with a non-zero status code if they are bad, you should either check for it or `set -e` so the entrypoint exits immediately.